tests/test_create_conference.py

```diff
@@ -0,0 +1,339 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+import os
+import datetime
+import openreview
+import pytest
+import time
+import json
+
+os.environ["OPENREVIEW_USERNAME"] = "OpenReview.net"
+os.environ["OPENREVIEW_PASSWORD"] = "1234"
+
+class TestConference():
+    @pytest.fixture(scope="class")
+    def conference(self, client):
+        pc_client=openreview.Client(username='pc@neurips.cc', password='1234')
+        request_form=client.get_notes(invitation='openreview.net/Support/-/Request_Form', sort='tmdate')[0]
+
+        conference=openreview.helpers.get_conference(pc_client, request_form.id)
+        return conference
+
+    def test_create_client(self, client, openreview_client):
+        # Test connectivity to the API
+
+        client = openreview.Client()
```

Comment:
why do you need to do this? `client` and `openreview_client` should be already created.